### DISCUSSION TITLE
Updated TIVO PF v3 remote html

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#538] [General]
Here is the latest html that I use.  I added a toggle button at the bottom to switch from the mapped TIVO remote image versi
on to the non-mapped HTML version.  You will need to customize your channels in the code.  There is a basic readme file in the zip file.
[TIVOremotePFv3.zip](https://github.com/GerryDazoo/Slinger/files/10775083/TIVOremotePFv3.zip)

**Comment by Alimond52** [comment]
pfuller88, thank you so much for this remote.
I have an old **Tivo** cable box with a built-in **HDD**. It is manufactured by **Samsung** GX-CM700CF/COM.
The remote is similar to the one you made. And the buttons on your remote control, can control almost every function on my Tivo box.
The only problem is that I would need to **reorganize** what function your buttons are associated with. I have read your Readme file. Understand some of what you write.
I will be using the new **Slinger Player on a Windows 10 PC**.
Can I edit your files in a regular **Text editor**?
Some of your buttons, I'm not sure which function they are connected to. Do you have any info that can **clarify** that?
This is what my remote control looks like.
I'm going abroad soon and staying for a long time. Would be fun if it would work with your nice Remote Control.

**Reply by pfuller88** [reply]
Hi Alimond52,
Yes you should be able to edit with a text editor like notepad++.   Below is a line of code from the file.  If you look at each line, the "title" tells you what button it is.  The number in the brackets next to "sendkeys" is the button code sent to your slingbox.   You can change the number or the title on each line as you see fit.

<area alt="input" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Input" onClick="sendKeys('22', 'input'); return false;" href="#" coords="47,130,17" shape="circle">
                <area alt="info" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Info" onClick="sendKeys('46', 'info'); return false;" href="#" coords="214,129,18" shape="circle">
                <area alt="back" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Back" onClick="sendKeys('37', 'back'); return false;" href="#" coords="40,176,21" shape="circle">

**Reply by pfuller88** [reply]
`<area alt="input" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Input" onClick="sendKeys('22', 'input'); return false;" href="#" coords="47,130,17" shape="circle">`

**Reply by Alimond52** [reply]
You are so nice! Thanks for help!!!!

**Comment by Alimond52** [comment]
![Tivo](https://user-images.githubusercontent.com/123761450/222970193-82f79021-4225-4e9d-b4dd-77e2c3a3b997.jpg)
